app/actions/todos.go

```diff
@@ -20,3 +22,30 @@ func Index(c buffalo.Context) error {
 
 	return c.Render(http.StatusOK, r.HTML("/todos/index.plush.html"))
 }
+
+func NewTask(c buffalo.Context) error {
+	todo := models.Todo{}
+	c.Set("todo", todo)
+
+	return c.Render(http.StatusOK, r.HTML("todos/new.plush.html"))
+}
+
+func PostNewTask(c buffalo.Context) error {
```

Comment:
It is our convention to use Save whenever we create a new resource.
Then, it should be Save instead of NewTask.